introduction.md

```diff
@@ -1,2 +1,7 @@
 Hi there! I'm Marc Pampols, a developer based in Spain.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?